element-template-generator/core/src/main/java/io/camunda/connector/generator/api/CliCompatibleTemplateGenerator.java

```diff
@@ -50,4 +53,52 @@ record ScanResult(
       Integer templateVersion,
       String templateType,
       Object additionalData) {}
+
+  /**
+   * Preview of the scanned operations result.
+   *
+   * @param id ID of the operation
+   * @param path Url path of the operation
+   * @param method Method(Get, Post) of the operation
+   * @param tags Tags of the operation
+   * @param supported Indicate whether the operation is supported or not
+   */
+  record Operation(String id, String path, String method, List<String> tags, boolean supported) {
+    public static class Builder {
+      String id;
+      String path;
+      String method;
+      List<String> tags;
+      boolean supported;
```

Comment:
@markfarkas-camunda would it be possible to add a `description` field?  Maybe its not used in the UI yet but it could be handy in the future. Both formats provide this field.

Reply:
Good idea! Added that!